utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
+% Copyright 2023 The MathWorks, Inc.
+prjFileName="fsdaToolboxPackaging.prj";
 toolboxOption=matlab.addons.toolbox.ToolboxOptions(prjFileName);
 
 % Update the version number
```

Comment:
Follow the MathWorks standard for variable naming, you cannot use abbreviation in camel cased full name. You can make the name more intuitive by using something along the lines of toolboxVersion. "Number" is an empty word.
verNumber input needs to be validated.